thirdeye-ui/src/app/components/rca/heat-map-section/heat-map-section.component.tsx

```diff
@@ -0,0 +1,260 @@
+/*
+ * Copyright 2023 StarTree Inc
+ *
+ * Licensed under the StarTree Community License (the "License"); you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at http://www.startree.ai/legal/startree-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the
+ * License is distributed on an "AS IS" BASIS, WITHOUT * WARRANTIES OF ANY KIND,
+ * either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+import { Box, Button, Grid, Typography } from "@material-ui/core";
```

Comment:
This is the renamed https://github.com/startreedata/thirdeye/pull/1360/files#diff-1c98ced186fa5a374cca0cca6048442290b48804c52809d50519fe691e157581 file that is shown as deleted below